deploy/cloudformation/static-host-pipeline.yml

```diff
@@ -14,6 +14,14 @@ Parameters:
     Type: String
     Description: An e-mail address of someone who can approve test environment
 
+  ApiURL:
```

Comment:
You already have SourceRepoOwner and SourceRepoName as a parameter. You could construct this ApiURL from those, since it's unlikely the github API endpoint will change.

Reply:
Are we making an assumption here that the repository will always be in GitHub? @jeremyf could respond in context of Mellon.

Reply:
I think we have to for the moment, since the code that performs the POST is still going to be based on their status API